### ISSUE TITLE
Validate html error when enable critical css

### ISSUE THREAD

**Issue by minhluan259** [magento/magento2#26760]
### Preconditions (*)

1. Magento 2.3.4

### Steps to reproduce (*)

1. Enable Critial CSS
2. Validate html https://validator.w3.org/

### Expected result (*)

1. Validate html without error

### Actual result (*)

1. Validate html error
https://prnt.sc/qz5dz9

https://github.com/magento/magento2/blob/9544fb243d5848a497d4ea7b88e08609376ac39e/app/code/Magento/Theme/Controller/Result/AsyncCssPlugin.php#L60-L66

need change to 
```
                    $loadCssAsync = sprintf(
                        '<link rel="preload" as="style" media="%s" 
                         onload="this.onload=null;this.rel=\'stylesheet\'"' .
                        ' href="%s">',
                        $media,
                        $href
                    );
```

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @sudheers-kensium 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-31378` were created

**Issue Available**: @sudheers-kensium, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Hi @minhluan259. Thank you for your report. 
The issue has been fixed in magento/magento2#26764 by @srsathish92 in 2.4-develop branch
Related commit(s):
 - [f9697f509ae09f5c9e0f0d7b1a2de7ed9a53fc16](https://github.com/magento/magento2/commit/f9697f509ae09f5c9e0f0d7b1a2de7ed9a53fc16)
 - [bf21c4bea3cbd600ee1333186bf43b3065847681](https://github.com/magento/magento2/commit/bf21c4bea3cbd600ee1333186bf43b3065847681)
 - [2881e4886fee787e5ecdf73edef4a197321bea43](https://github.com/magento/magento2/commit/2881e4886fee787e5ecdf73edef4a197321bea43)
 - [1e50859a6c2f52984a9a2812fb6c93aa6b0a0f34](https://github.com/magento/magento2/commit/1e50859a6c2f52984a9a2812fb6c93aa6b0a0f34)

The fix will be available with the upcoming 2.4.0 release.
